### PR TITLE
BDD test spawn command

### PR DESCRIPTION
**Pull request by phansch**
This PR adds integration tests for the `:spawn` command :sparkles:

Unfortunately I can't get the userscript test to work as it can't find the script with the relative path:
```
guiprocess:_pre_start:138 Executing: /tmp/qutebrowser-basedir-8rqezy_7/data/userscripts/open_current_url
ipc:on_error:234 Socket 0x7f416a915ca8: error 1: QLocalSocket: Remote closed
```

It works if I use an absolute path.
I guess the new userscripts folder does not get included for the test environment, but I'm not sure how to fix it.

Also if you have ideas for more tests, let me know :)

### CODE REVIEW

**Comment by The-Compiler** [comment]
I'd say use an absolute path, and add a new `(datapath)` replacement to `run_command` in `tests/integration/conftest.py` which turns into `/path/to/qutebrowser/tests/integration/data` (using `os.path.join`, `os.path.abspath` and `__file__`).

File: tests/integration/features/spawn.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    Scenario: Running :spawn
+        When I run :spawn -v echo "Hello"
+        Then the message "Command exited successfully." should be shown
+
+    Scenario: Running :spawn with command that does not exist
+        When I run :spawn command_does_not_exist127623
+        Then the error "Error while spawning command: The process failed to start." should be shown
+
+    Scenario: Running :spawn with invalid quoting
+        When I run :spawn """
```
Comment (The-Compiler) [inline tests/integration/features/spawn.feature]: I think you could use `"'"` here to avoid confusing syntax highlighting

**Comment by phansch** [comment]
It looks like `apt-get update` failed in one of the travis builds. Do you want me to rebase anything?

**Comment by The-Compiler** [comment]
It's just Travis' network connection being funny sometimes. I'll restart it.

**Comment by The-Compiler** [comment]
Thank you! The failing checks were unrelated to your changes.

**Pull request merged**